libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -271,6 +271,15 @@ class MapboxNavigation(
      */
     val historyRecorder = MapboxHistoryRecorder(navigationOptions, logger)
 
+    /**
+     * Navigation native experimental API. Do not store a link to [Experimental] object, use it with
+     * `mapboxNavigation.experimental` because [MapboxNavigation] is responsible for it's lifecycle.
+     *
+     * Add a separate gradle dependency to use it
+     * "com.mapbox.navigator:mapbox-navigation-native:x.y.z"
```

Comment:
let's remove this. We can communicate to the interested parties what dependency should be used when necessary.